debian-packaging/package-user-root/exam-engine/bin/start-mex

```diff
@@ -4,4 +4,4 @@
 EXAM_ENGINE_DIR=$(realpath ~/exam-engine/)
 nvm use "$(cat "$EXAM_ENGINE_DIR"/.nvmrc)"
 
-"$EXAM_ENGINE_DIR"/scripts/start.sh koe.xml
+"$EXAM_ENGINE_DIR"/packages/cli/dist/start.js koe.xml
```

Comment:
hmm, voiko tämän tsc:n(?) generoiman .js-tiedoston ajaa vaan tälleen shell-skriptistä?

Reply:
Will have to add node there to make it work. Or add a ’#!/usr/bin/env node’ shebang line to the CLI scripts.

Reply:
Added shebangs to all CLI scripts

Reply:
Added shebangs to all CLI scripts